### ISSUE TITLE
Try to pretrain on detection dataset

### ISSUE THREAD

**Issue by merlinarer** [kakaobrain/scrl#8]
Hello, Thanks for sharing your codes ~
I try to train on VOC2012 without label to get the pretrained model, and then finetune with VOC2012 detection label. The result is very poor, but the result should be better Since pretrain dataset and finetune dataset is the same. Did you tried this or do you have any idea that what's the reason?

**Comment by merlinarer** [comment]
> Hello, Thanks for sharing your codes ~
> I try to train on VOC2012 without label to get the pretrained model, and then finetune with VOC2012 detection label. The result is very poor, but the result should be better Since pretrain dataset and finetune dataset is the same. Did you tried this or do you have any idea that what's the reason?

Ohh~ I'm sorry, I saw the experiment with the same setting in the appendix just now that upstream is trained with the unlabeled COCO dataset. Could you please provide the cfg file, log and checkpoints in this experiment?  thanks a lot ^^

**Comment by prayforher** [comment]
As we described in the paper, you can use the same configuration file except for the training epoch and training data path for COCO/VOC dataset.
However, I'm concerned that the number of images for VOC2012 trainval is not sufficient to train the upstream model.

**Comment by merlinarer** [comment]
``

> As we described in the paper, you can use the same configuration file except for the training epoch and training data path for COCO/VOC dataset.
> However, I'm concerned that the number of images for VOC2012 trainval is not sufficient to train the upstream model.

Thanks, I notice that there are 2 cfg file (scrl_200ep and  scrl_1000ep) with different hyper params like lr and momentum, and which one did you use to train on COCO for 2000 epochs?  Would you please provide the log and checkpoints?
